maven/await-artifact/README.md

```diff
@@ -12,11 +12,12 @@ NOTE: this action does not timeout, hence you need to configure your GitHub work
 
 ## Inputs
 <!--inputs-->
-| Name          | Description                         | Required | Default |
-|---------------|-------------------------------------|----------|---------|
-| `artifact-id` | Maven artifact-ID of the artifact   | `true`   | ` `     |
-| `group-id`    | Maven group-ID of the artifact      | `true`   | ` `     |
-| `version`     | Version of the artifact to wait for | `true`   | ` `     |
+| Name                     | Description                                                                       | Required | Default |
+|--------------------------|-----------------------------------------------------------------------------------|----------|---------|
+| `artifact-id`            | Maven artifact-ID of the artifact                                                 | `true`   | ` `     |
+| `group-id`               | Maven group-ID of the artifact                                                    | `true`   | ` `     |
+| `version`                | Version of the artifact to wait for                                               | `true`   | ` `     |
+| `wait-for-maven-central` | Whether to wait for the artifact to be available in the maven central repository. | `false`  | `false` |
```

Comment:
maybe add as footnote that when set to false, only the sonatype repository is checked, which is usually fine for snapshots, but for releases checking maven central might be preferable as the sonatype proxy repo might not reflect actual maven central state.

Reply:
Sure thing

For the record, I decided to use this approach to avoid a breaking change since it's a change of behaviour

Reply:
what do you think with the new changes? https://github.com/elastic/oblt-actions/pull/133/commits/61f4cc51932e737b24dabb32c29a99320868b968

I added a longer description and also a new flag to support whether to validate sonatype or maven or both.

Reply:
Looks good to me, adding a sentence to explain when to change the default for `wait-for-maven-central` might be welcome:
- with default value, we just wait for the publication to complete on sonatype, but the actual availability in maven central might not be ready yet.
- with `true` we wait for the artifact to be published in maven central, this should be used when building other artifacts by downloading from maven central, which is for example quite common for docker images.

Reply:
See [`e695a48` (#133)](https://github.com/elastic/oblt-actions/pull/133/commits/e695a4856bd20d3fd7e911a13bdeae65f03fda79)